README.rst

```diff
@@ -465,11 +465,12 @@ where:
 
 For example, consider::
 
-     schedule_suite.sh rbd wip-fix cuttlefish bob.smith@foo.com master cuttlefish plana
+     teuthology-suite -s rbd -c wip-fix -k cuttlefish -e bob.smith@foo.com -f master -t cuttlefish -m plana
 
 The above command runs the rbd suite using wip-fix as the ceph branch, a
 straight cuttlefish kernel, and the master flavor of cuttlefish teuthology.  It
```

Comment:
I don't think this command is defined quite right.  The -f, or --flavor, is the kernel flavor you want to run against.  This should be "basic", "gcov" or 'notmalloc'.

Here's a proposed revision:

"The above command runs the rbd suite using the wip-fix branch of ceph, the cuttlefish kernel branch, ran against a 'basic' kernel flavor and the teuthology cuttlefish branch will be used.  This will run on plana machines and send an email to bob.smith@foo.com when it's complete."